README.md

```diff
@@ -91,16 +91,34 @@ Run `yarn clean-all` if the `yarn start` command fails due to a change in the co
 - `yarn grain` distributes Grain according to the current Cred scores, and the config in `config/grain.json`. 
 
 This repo also contains a GitHub action for automatically distributing grain. It will run every Sunday and create a Pull Request
-with the ledger updated with the new grain balances based on the users cred scores. The amount of grain to get distributed
-every week can be defined in the `config/grain.json` file. There are two different policies that can be used to control
+with the ledger updated with the new grain balances based on the users Cred scores. The amount of grain to get distributed
+every week can be defined in the `config/grain.json` file. 
```

Comment:
This should be after descriptions of the policies IMO.  Start with a high level picture of distributing grain, and then go into details of how it works.